.gitignore

```diff
@@ -1,5 +1,6 @@
 # Output directory
 output
+logilica-cli/*
```

Comment:
This doesn't seem generally applicable, so I would recommend against this change.

(Also, I'm pretty sure that the asterisk is superfluous.)

Reply:
FWIW, the existing `.gitignore` file has an `Environments` section which includes many of the typical names used for venv's.  If you use one of those instead of `logilica-cli`, then you'll be covered already.  (I typically use `.venv` so that `ls` doesn't show it.)

Reply:
Pushed a new commit that should have removed the change from `.gitignore`. Good to know that some common names are already excluded if I were to update, thanks.